designs/administrator/4.6/column-management/index.md

```diff
@@ -0,0 +1,53 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Column Management
+
+## Behavior
+Column management will live in the toolbar as a column icon on the Pods page on desktop. It will be hidden on mobile. Clicking on the icon will open a modal that lists all column names with checkboxes.
+![column-management-toolbar](img/column-management-toolbar.png)
+If the user is looking at “all projects”, then the checkbox next to “name” as well as “namespace”  in the modal will be shown as checked and disabled. Hovering over it will bring up a tooltip that explains that the columns may not be edited.
+![column-management-disabled-check](img/column-management-disabled-check.png)
+The modal will have an informational alert by default that reads “You can select up to 7 columns”. If 7 columns have already been selected, then all checkboxes will be disabled. Once the user deselects a checkbox, the other checkboxes will return to default state (rather than disabled). 
+![column-management-check](img/column-management-check.png)
```

Comment:
nit: Should it say `Default Pod Columns` using `Pods` sounds weird to me

Reply:
@matthewcarleton I don't exactly remember why we ended up going that way - I think because it made more sense when referring to that specific table and comparing it to "Additional columns"... there was definitely a lot of discussion around it which led us there

Reply:
I'll trust the conclusion then :)